Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:jessie-slim
+ARG bkup_version=2.11.1
```

Comment:
Should we download the zip from the stable branch (https://github.com/github/backup-utils/archive/stable.zip) so we don't have to bump this manually?

Reply:
We can do that. It is a little wonky in that we still need to `COPY` in the `ghe-docker-init` file from the repo so that the container has the latest changes to that file (especially at this point since `stable` containtain the file yet). New commit incoming.